### PR TITLE
[RHPAM-3725] - Missing RHPAM Kogito operator description in OperatorH…

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
…ub OCP page

Jira Issue : https://issues.redhat.com/browse/RHPAM-3725

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [x] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Comment by vaibhavjainwiz** [comment]
New Description looks like as below:
Screenshot from 2021-06-15 19-17-14

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -31,11 +31,14 @@ metadata:
       ]
     capabilities: Basic Install
     categories: Integration & Delivery
+    certified: "true"
     containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:7.11.0
+    description: RMPAM Kogito Operator for deployment and management of Kogito services.
```
Comment (sutaakar) [inline bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:36]:
```suggestion
    description: RHPAM Kogito Operator for deployment and management of Kogito services.
```

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -52,7 +55,15 @@ spec:
       kind: KogitoRuntime
       name: kogitoruntimes.rhpam.kiegroup.org
       version: v1
-  description: RMPAM Kogito Operator is designed to deploy Kogito services.
+  description: |-
+    RMPAM Kogito Operator is designed for deployment and management of Kogito services from source (only on OpenShift) and every piece of infrastructure that the services might need:
```
Comment (sutaakar) [inline bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:59]:
```suggestion
    RHPAM Kogito Operator is designed for deployment and management of Kogito services from source (only on OpenShift) and every piece of infrastructure that the services might need:
```

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -5,11 +5,14 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     categories: Integration & Delivery
+    certified: "true"
     containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:7.11.0
+    description: RMPAM Kogito Operator for deployment and management of Kogito services.
```
Comment (sutaakar) [inline config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:10]:
```suggestion
    description: RHPAM Kogito Operator for deployment and management of Kogito services.
```

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -26,7 +29,15 @@ spec:
       kind: KogitoBuild
       name: kogitobuilds.rhpam.kiegroup.org
       version: v1
-  description: RMPAM Kogito Operator is designed to deploy Kogito services.
+  description: |-
+    RMPAM Kogito Operator is designed for deployment and management of Kogito services from source (only on OpenShift) and every piece of infrastructure that the services might need:
```
Comment (sutaakar) [inline config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:33]:
```suggestion
    RHPAM Kogito Operator is designed for deployment and management of Kogito services from source (only on OpenShift) and every piece of infrastructure that the services might need:
```

**Review by sutaakar — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (vaibhavjainwiz) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:36]: done

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (vaibhavjainwiz) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:59]: done

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (vaibhavjainwiz) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:10]: done

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 4 above)
Comment (vaibhavjainwiz) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:33]: done

**Review by radtriste — APPROVED** [review]

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -52,7 +55,15 @@ spec:
...
+
+    * Builds a Kogito service from a remote git repository and creates a custom image for the given runtime: Quarkus or Springboot. [See more](https://docs.jboss.org/kogito/release/latest/html_single/#con-kogito-operator-deployment-options_kogito-deploying-on-openshift).
```
[the remark this answers is not in the document]
Comment (vaibhavjainwiz) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:61]: done

[the remark this answers is not in the document]
Comment (vaibhavjainwiz) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:61]: done

[the remark this answers is not in the document]
Comment (vaibhavjainwiz) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:61]: @spolti How could help us to confirm it?

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 6]
```diff
@@ -26,7 +29,15 @@ spec:
...
+
+    * Builds a Kogito service from a remote git repository and creates a custom image for the given runtime: Quarkus or Springboot. [See more](https://docs.jboss.org/kogito/release/latest/html_single/#con-kogito-operator-deployment-options_kogito-deploying-on-openshift).
```
[the remark this answers is not in the document]
Comment (vaibhavjainwiz) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:35]: done

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 7]
```diff
@@ -52,7 +55,15 @@ spec:
...
+    * Deploy Kogito service from user provided image.
```
[the remark this answers is not in the document]
Comment (vaibhavjainwiz) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:62]: done

File: config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 8]
```diff
@@ -26,7 +29,15 @@ spec:
...
+    * Deploy Kogito service from user provided image.
```
[the remark this answers is not in the document]
Comment (vaibhavjainwiz) [reply config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml:36]: done

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (spolti) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml]: I am trying to confirm, also, @evacchi probably knows :)

[the remark this answers is not in the document]
Comment (spolti) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml]: just confirmed it here, that's correct, the productized Quarkus name is "Red Hat Build of Quarkus"

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
please cherry pick to branch 1.5.x as well.

**Comment by vaibhavjainwiz** [comment]
raised cheery pick request to 1.5.x
https://github.com/kiegroup/rhpam-kogito-operator/pull/46

**Pull request merged**